### PR TITLE
html-forms-Laura Grace Manimtim

### PR DESCRIPTION
**Pull request by lauragracem**
Screen Shot 2022-05-10 at 12 03 27 PM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div class="text">Email</div>
```
Comment (Zhiwutian) [inline html-forms/index.html]: You want the `Email` text to be within the label. That is to create the functionality of being able to click on the label and activate the input

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <div>
+        <label for="email"></label>
+        <input type="text" name="email" size="12" />
+      </div>
+      <div>Why is coding cool?</div>
```
Comment (Zhiwutian) [inline html-forms/index.html]: This should be a label for the same reason.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <div>
+        <select name="Select an option">
+          <option value="cool">Because it's cool</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <div>
+        <input type="radio" name="language" value="html" />HTML
```
Comment (Zhiwutian) [inline html-forms/index.html]: each of these input should have a label, you can wrap the label around the input if desired. That will maintain the click functionality

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <input type="radio" name="language" value="css" />CSS
+        <input type="radio" name="language" value="javascript" />JavaScript
+      </div>
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" name="sign up" value="yes"/>Sign me up!
+      <div class="text">Comments</div>
```
Comment (Zhiwutian) [inline html-forms/index.html]: you want a label here as well

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
make sure that all inputs have associated labels

File: html-forms/index.html (hunk 4 above)
Comment (lauragracem) [reply html-forms/index.html]: Screen Shot 2022-05-10 at 12 36 38 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**